pkg/reconciler/instances/scmigration/scmigration.go

```diff
@@ -0,0 +1,22 @@
+package scmigration
+
+import (
+	"github.com/kyma-incubator/reconciler/pkg/logger"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/service"
+)
+
+const ReconcilerName = "sc-migration"
+
+//nolint:gochecknoinits
+func init() {
+	log := logger.NewLogger(false)
+
+	log.Debugf("Initializing component reconciler '%s'", ReconcilerName)
+	reconciler, err := service.NewComponentReconciler(ReconcilerName)
+	if err != nil {
+		log.Fatalf("Could not create '%s' component reconciler: %s", ReconcilerName, err)
+	}
+
+	reconciler.WithDependencies("btp-operator")
```

Comment:
Please drop this call as we are going to remove it from reconciler soon (we will add a deprectation mark to it soon).

Please implement your reconciler in way, that it is able to identify such dependencies by its own or can be fixed by retrying it until the dependency is in place.